.github/workflows/build.yml

```
@@ -0,0 +1,30 @@
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**" # Will still build PRs, but not the branches themself
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)